### ISSUE TITLE
Contradictory Information Regarding Windows 10/11 Enterprise LTSC Support for WUfB

### ISSUE THREAD

**Issue by jonconwayuk** [MicrosoftDocs/memdocs#3992]
Windows Update for Business supports Windows 10/11 Enterprise LTSC which contradicts the informational note in this article:

https://learn.microsoft.com/en-us/mem/intune/protect/windows-driver-updates-overview#device--edition-requirements

Note
Unsupported versions and editions:
Windows 10/11 Enterprise LTSC: Windows Update for Business (WUfB) does not support the Long Term Service Channel release. Plan to use alternative patching methods, like WSUS or Configuration Manager.

Contradicting information available here:

https://learn.microsoft.com/en-us/windows/deployment/update/waas-manage-updates-wufb?branch=pr-en-us-8343

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7928e48c-63f2-e459-db87-bb62eb5501b0
* Version Independent ID: 7928e48c-63f2-e459-db87-bb62eb5501b0
* Content: [Learn about Windows Driver updates policy for Windows 10 Windows 11 devices in Intune](https://learn.microsoft.com/en-us/mem/intune/protect/windows-driver-updates-overview)
* Content Source: [memdocs/intune/protect/windows-driver-updates-overview.md](https://github.com/MicrosoftDocs/memdocs/blob/main/memdocs/intune/protect/windows-driver-updates-overview.md)
* Service: **microsoft-intune**
* Sub-service: **protect**
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by msbemba** [comment]
@jonconwayuk Thank you for your feedback. The document was recently updated and the statement "Windows 10/11 Enterprise LTSC: Windows Update for Business (WUfB) does not support the Long Term Service Channel release. Plan to use alternative patching methods, like WSUS or Configuration Manager." is correct. 

We will check if the other article -https://learn.microsoft.com/en-us/windows/deployment/update/waas-manage-updates-wufb?branch=pr-en-us-8343 can be updated.

Hope this helps!

Thanks
Sri

**Comment by jonconwayuk** [comment]
As far as I am aware WUfB should now support Windows 10/11 IoT Enterprise and Windows 10/11 IoT Enterprise LTSC after it was requested via the Product Team. 

This was updated on the Docs a couple of weeks ago after support was confirmed.

I think the question that remains is around _WUfB Deployment Services_ and whether it supports IoT and LTSC. I have raised this question with Lorena Mora who is going to speak to the Windows Servicing team to see what the situation is.
